speed-dreams/source-2.2.3/data/tracks/curved-highway/curved-highway-small-4-80/curved-highway-small-4-80.xml

```diff
@@ -166,6 +167,7 @@ the Free Software Foundation; either version 2 of the License, or
       <!--******************************-->
       <section name="curve 1">
         <attstr name="type" val="rgt" />
+		  <attnum name="speedLimit" val ="80" />
```

Comment:
remove tabs

Reply:
forgot to change tabs in Cas's IDE, will be changed soon